internal/runners/shell/shell.go

```diff
@@ -81,6 +76,7 @@ func (u *Shell) Run(params *Params) error {
 		return locale.WrapInputError(err, "err_shell_runtime_new", "Could not start a shell/prompt for this project.")
 	}
 
+	fmt.Printf(`rti: %v, proj: %v`, rti, proj)
```

Comment:
Debug? If so, you might want to scan for more of these since I'm not digging to hard. Or should I do the scan?

Reply:
Will do a scan, thanks for picking up on these.